### PR TITLE
Add bootstrap

### PR DESCRIPTION
**Pull request by tisuchi**
- task 1
- table

### CODE REVIEW

File: web.html [hunk 1]
```diff
@@ -2,38 +2,44 @@
 <html lang="en">
 <head>
 	<meta charset="UTF-8">
-	<title>Welcome Page</title>
+	<title>Suchi</title>
+
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
```
Comment (minhaislam) [inline web.html:8]: Why did you add this?

Comment (tisuchi) [reply web.html:8]: Because we need this.

File: web.html [hunk 2]
```diff
@@ -2,38 +2,44 @@
...
+		</tr>
+		<tr>
+			<td>02</td>
+			<td>Student Name</td>
+		</tr>
 		<tr>
-			<td></td>
-			<td></td>
-			<td></td>
+			<td>02</td>
```
Comment (minhaislam) [inline web.html:23]: ID should be 3 now and it will be incrementatl.
